webapp/src/tutorial.tsx

```diff
@@ -232,7 +245,7 @@ export class TutorialCard extends data.Component<ISettingsProps, {}> {
                     <div className={`tutorialmessage ${hasHint ? 'focused' : undefined}`} role="alert" aria-label={tutorialAriaLabel} tabIndex={hasHint ? 0 : -1} onClick={() => {if (hasHint) this.showHint();}} onKeyDown={sui.fireClickOnEnter}>
                         <div className="content" dangerouslySetInnerHTML={{__html: tutorialHeaderContent}} />
                     </div>
-                    <sui.Button class="large green okbutton showlightbox focused" text={lf("Ok") } onClick={() => this.closeLightbox() } onKeyDown={sui.fireClickOnEnter} />
+                    <sui.Button id="okButton" class="large green okbutton showlightbox focused" text={lf("Ok") } onClick={() => this.closeLightbox() } onKeyDown={sui.fireClickOnEnter} />
```

Comment:
Right, furthermore we might have several okButton too in the future which would be problematic. I'll edit it.